src/breadcrumbs/breadcrumbs.macros.html

```diff
@@ -24,21 +23,25 @@
         </a>
       </li>
       <li class="breadcrumbs-list-item">
-        {{ icon('arrow', size='tiny') }}
+        >
```

Comment:
If we're going to use a character, it would be best to use the unicode characters for these symbols to ensure we're okay when we localize.